### PR TITLE
XEH - Fix missing XEH on AoW objects (2.02)

### PR DESCRIPTION
**Pull request by PabstMirror**

### CODE REVIEW

**Comment by commy2** [comment]
> GalleryDioramaUnit

Huh?

File: addons/xeh/CfgVehicles.hpp [hunk 1]
```diff
@@ -397,4 +397,9 @@ class CfgVehicles {
     class Land_MapBoard_01_Map_Syndikat_Tanoa_F: Land_Map_blank_F {
         XEH_ENABLED;
     };
+
+    // AOW (2.02)
+    class GalleryDioramaUnit_01_base_F: Items_base_F {
+        XEH_ENABLED;
+    };
```
Comment (commy2) [inline addons/xeh/CfgVehicles.hpp:404]: This change does not necessarily require 2.02, but I guess it is a good thing to update this anyway? Would merge the PR now if the requirement was not set.

Comment (PabstMirror) [reply addons/xeh/CfgVehicles.hpp:404]: This is based on current RC and sometimes things change, so we shouldn't merge until 2.02 is officially released

Comment (PabstMirror) [reply addons/xeh/CfgVehicles.hpp:404]: still needed in release

**Review by commy2 — APPROVED** [review]

**Pull request merged**